recipe/activate-gfortran.sh

```diff
@@ -84,7 +84,7 @@ function _tc_activation() {
 # When people are using conda-build, assume that adding rpath during build, and pointing at
 #    the host env's includes and libs is helpful default behavior
 if [ "${CONDA_BUILD:-0}" = "1" ]; then
-  FFLAGS_USED="@FFLAGS@ -I${PREFIX}/include -fdebug-prefix-map=\${SRC_DIR}=/usr/local/src/conda/\${PKG_NAME}-\${PKG_VERSION} -fdebug-prefix-map=\${PREFIX}=/usr/local/src/conda-prefix"
+  FFLAGS_USED="@FFLAGS@ -Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib -I${PREFIX}/include -fdebug-prefix-map=\${SRC_DIR}=/usr/local/src/conda/\${PKG_NAME}-\${PKG_VERSION} -fdebug-prefix-map=\${PREFIX}=/usr/local/src/conda-prefix"
```

Comment:
You need to add a new variable, `LDFLAGS_USED` and copy https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/master/recipe/activate-clang.sh#L89-L90
and https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/master/recipe/activate-clang.sh#L89-L90
and https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/master/recipe/activate-clang.sh#L89-L90

Also needs to copy these to the deactivation script.

Reply:
I guess you copied-and-pasted the same link 3 times :smile:
Can you take a second look?

PS: I'm kind of lost on what needs to be done in the deactivation.

Reply:
Looks good to me. deactivation needs to do the same thing. Here's an example, https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/master/recipe/deactivate-clang.sh#L89-L90

Reply:
Thanks! I was confused b/c the deactivation script here did not had that if-clause. Hope I got it right in the last commit.